tree/ntuple/v7/test/ntuple_packing.cxx

```diff
@@ -211,6 +213,7 @@ TEST(Packing, OnDiskEncoding)
       *e->Get<float>("float") = std::nextafterf(1.f, 2.f); // 0 01111111 00000000000000000000001 == 0x3f800001
       *e->Get<double>("double") = std::nextafter(1., 2.);  // 0x3ff0 0000 0000 0001
       *e->Get<ClusterSize_t>("index32") = 39916801;        // 0x0261 1501
+      *e->Get<ClusterSize_t>("index64") = 39916801;        // 0x0261 1501
```

Comment:
Perhaps we should use a value that has some non-zero bits for bits $32-63$ (also below in line 227).